chia/cmds/wallet_funcs.py

```diff
@@ -799,3 +799,16 @@ async def list_nfts(args: Dict, wallet_client: WalletRpcClient, fingerprint: int
             print(f"No NFTs found for wallet with id {wallet_id} on key {fingerprint}")
     except Exception as e:
         print(f"Failed to list NFTs for wallet with id {wallet_id} on key {fingerprint}: {e}")
+
+
+async def set_nft_did(args: Dict, wallet_client: WalletRpcClient, fingerprint: int) -> None:
+    wallet_id = args["wallet_id"]
+    did_id = args["did_id"]
+    nft_coin_id = args["nft_coin_id"]
+    fee = args["fee"]
+    try:
+        response = await wallet_client.set_nft_did(wallet_id, did_id, nft_coin_id, fee)
+        spend_bundle = response["spend_bundle"]
+        print(f"DID set on NFT successfully with spend bundle: {spend_bundle}")
```

Comment:
Technically we pushed that spend bundle to mempool and it's going to get processed at some point, but it's NFT hasn't be updated yet at this point. 
We could provide the user with command to check status of the transaction here? Similar to how `send_transaction` does it.

Reply:
That would require altering the API as it currently only returns the spend bundle. Should we change the wording here to indicate this spend bundle was sent? or?

Reply:
ok, lets change the wording for now

Reply:
Sent a different wording.